src/main/java/se/nackademin/Personell.java

```diff
@@ -0,0 +1,49 @@
+package se.nackademin;
+
+class Personell {
+	private String name;
+	private int age;
+	private String title;
+	private int salary; // per annum
+	private int balance;
+	
+	public String getName() { return this.name; }
+	public int getAge() { return this.age; }
+	public String getTitle() { return this.title; }
+	public int getSalary() { return this.salary; }
+	public int getBalance() { return this.balance; }
+
+	public boolean sell(boolean isValid) {
```

Comment:
Not sure on design intention here? "sell" means what for a personell? And what does isValid represent, what is or isn't valid? What's the difference between calling sell(false) and just not calling sell at all?